wagtail/admin/widgets/auto_height_text.py

```diff
@@ -5,11 +5,12 @@
 
 
 class AdminAutoHeightTextInput(widgets.Textarea):
-    template_name = "wagtailadmin/widgets/auto_height_text_input.html"
-
     def __init__(self, attrs=None):
         # Use more appropriate rows default, given autoheight will alter this anyway
-        default_attrs = {"rows": "1"}
+        default_attrs = {
+            "rows": "1",
+            "data-controller": "w-auto-size",
+        }
         if attrs:
             default_attrs.update(attrs)
 
```

Comment:
This file (below) still has all the Telepath instantiation (see lines 25 to 30), this is flagging an error in the console and I am pretty sure this can be removed.